third_party/move/move-binary-format/src/check_bounds.rs

```diff
@@ -115,6 +114,7 @@ impl<'a> BoundsChecker<'a> {
             self.view.struct_variant_instantiations(),
             Self::check_struct_variant_instantiation,
         )?;
+        self.check_function_defs()?;
```

Comment:
nit: `// Since bytecode version 7` comments (and similar ones) are no longer valid if we re-order, maybe remove them, or have an ending comment above.